bin/tronctl

```diff
@@ -29,13 +29,13 @@ COMMAND_HELP = (
     ('backfill', 'Print tronctl commands for starting many jobs for a particular date range'),
     ('disable', 'Disable selected job and cancel any outstanding runs'),
     ('enable', 'Enable the selected job and schedule the next run'),
-    ('fail', 'Mark an UNKNOWN job as having failed'),
-    ('success', 'Mark an UNKNOWN job as having succeeded'),
-    ('skip', 'Skip a failed action, runs dependent actions.'),
+    ('fail', 'Mark an UNKNOWN job as having failed. Does not trigger downstream jobs.'),
+    ('success', 'Mark an UNKNOWN or failed job or action as having succeeded. Will trigger downstream jobs.'),
```

Comment:
You can't move a job from failed to success, you can only skip failed jobs. So keep this as just UNKNOWN.